graspy/cluster/autogmm.py

```diff
@@ -17,12 +17,15 @@
 from sklearn.cluster import AgglomerativeClustering
 from sklearn.metrics import adjusted_rand_score
 from sklearn.mixture import GaussianMixture
-from sklearn.mixture.gaussian_mixture import (
+from sklearn.mixture._gaussian_mixture import (
     _compute_precision_cholesky,
     _estimate_gaussian_parameters,
 )
 from sklearn.model_selection import ParameterGrid
+from sklearn.utils._testing import ignore_warnings
+from sklearn.exceptions import ConvergenceWarning
 from joblib import Parallel, delayed
+import warnings
```

Comment:
@caseypw  is this import necessary

Reply:
Yes because in line 518 I throw a warning for the cosine similarity case. Did we still want a warning for that?

Reply:
ah ok that's fine